jpo-geojsonconverter/src/main/java/us/dot/its/jpo/geojsonconverter/converter/map/MapTopology.java

```diff
@@ -44,22 +44,30 @@ public static Topology build(String mapOdeJsonTopic, String processedMapTopic, M
             rawOdeMapStream.mapValues(
                 (Void key, Bytes value) -> {
                     DeserializedRawMap deserializedRawMap = new DeserializedRawMap();
-                    JsonValidatorResult validationResults = mapJsonValidator.validate(value.get());
-                    deserializedRawMap.setOdeMapOdeMapData(JsonSerdes.OdeMap().deserializer().deserialize(mapOdeJsonTopic, value.get()));
-                    deserializedRawMap.setValidatorResults(validationResults);
-                    logger.debug(validationResults.describeResults());
+                    try {
+                        JsonValidatorResult validationResults = mapJsonValidator.validate(value.get());
+                        deserializedRawMap.setOdeMapOdeMapData(JsonSerdes.OdeMap().deserializer().deserialize(mapOdeJsonTopic, value.get()));
+                        deserializedRawMap.setValidatorResults(validationResults);
+                        logger.debug(validationResults.describeResults());
+                    } catch (Exception e) {
+                        logger.error("Error in SpatValue:", e);
```

Comment:
Attach the error message to the DeserializedRawMap so downstream is aware there was an error.
Or if you intended to return null, do that instead of returning the empty DeserializedRawMap.

Reply:
Okay, John was concerned about downstream being able to handle a null value so that is why that was filtered out. I can work on sending an empty message, it might require some re-working on the processed topic generation.

Reply:
Don't do this and the previous comment, do one or the other.

Reply:
Yeah, I just noticed that there appears to be a bug in what is intended (intended to send null value, but it looks like it is sending empty messages instead). Either fix it to actually send the null value, or attach the error message so the DeserializaedRawMap isn't empty.

Reply:
OK I see in the MapProcessorJsonConverter code there is already a try-catch which would throw a null pointer exception when it received the empty DeserializedRawMap and return a null, which would be removed in the topology.  So it appears the code does function as stated.  I'd still recommend changing this so that the above step returns null just to make the error logs show a single error message rather than 2 errors in 2 steps.